main.tf

```diff
@@ -5,6 +5,7 @@ locals {
       export         = aws_efs_file_system.this.id
       license_secret = var.license_secret
       mount_point    = "/opt/nexus/sonatype-work"
+      extra_userdata = var.extra_userdata
```

Comment:
I hate to hold up the PR on this, but I think it's far better to do what we did here:

https://github.com/rhythmictech/terraform-aws-atlassian-bitbucket/blob/520770024410885e70a309ad6a8d27e677b0afc4/main.tf#L11https://github.com/rhythmictech/terraform-aws-atlassian-bitbucket/blob/520770024410885e70a309ad6a8d27e677b0afc4/main.tf#L11

Otherwise, you're making the assumption that the additional metadata is not only a bash script, but that it should have the same execution flags (e.g., `-e`).

Reply:
Thanks, I almost asked about this